src/rand_impls.rs

```diff
@@ -246,6 +246,12 @@ impl<T:Rand> Rand for Option<T> {
     }
 }
 
+impl<T: SeedableRng> Rand for T {
```

Comment:
Added a default implementation, so I could remove it from all PRNG's.

Reply:
I suppose this will get removed soon enough anyway.